linera-service/tests/end_to_end_tests.rs

```diff
@@ -516,7 +519,7 @@ async fn test_wasm_end_to_end_counter(config: impl LineraNetConfig) -> Result<()
 #[test_log::test(tokio::test)]
 async fn test_wasm_end_to_end_counter_publish_create(config: impl LineraNetConfig) -> Result<()> {
     use counter::CounterAbi;
-
+    #[cfg(not(feature = "kubernetes"))]
```

Comment:
Using `cfg()` here is not fully correct.
* Ideally, we should be able to run tests for an arbitrary subset of DBs at the same time, with all features activated:
`init-DB1; init-DB2; cargo test -p linera-service --features with_db1,with_db2; delete-DB1; delete-DB2`
* Would it be possible to use something like `config.database()` instead?
* If not, then consider creating a task and a TODO.

Reply:
That is clearly beyond the scope of this PR.

What I wanted with this PR is the following:
* Enable the Kubernetes end-to-end tests to run simultaneously.
* Make the `run_node_service` use different ports.

Therefore, I will create a task and TODO.